django/core/management/commands/inspectdb.py

```diff
@@ -116,14 +116,18 @@ def table2model(table_name):
                         extra_params['unique'] = True
 
                     if is_relation:
+                        rel_type = (
+                            'OneToOneField' if extra_params.pop('unique', False) or extra_params.get('primary_key')
+                            else 'ForeignKey'
+                        )
```

Comment:
I'm not sure if ternary fits here :thinking: , I would use
```python
if extra_params.pop('unique', False) or extra_params.get('primary_key'):
    rel_type = 'OneToOneField'
else:
    rel_type = 'ForeignKey'
```

Reply:
I used it because the definition of `rel_to` on the following line does that as well, but can/will change.